### PR TITLE
OSSMDOC-528 Distributed Tracing 2.4 Release Notes.

### PR DESCRIPTION
**Pull request by JStickler**
Version(s): 4.6 - 4.11

Issue: [OSSMDOC-528](https://issues.redhat.com/browse/OSSMDOC-528)
Documents [TRACING-1990](https://issues.redhat.com/browse/TRACING-1990)
Resolves [TRACING-2635](https://issues.redhat.com/browse/TRACING-2635)

Link to docs preview: http://file.bos.redhat.com/jstickle/OSSMDOC-528/distr_tracing/distributed-tracing-release-notes.html

<strike>https://deploy-preview-45350--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html</strike>

Eng review - frzifus, pavollloffay
QE review - iblancasa
Peer review - jc-berger

### CODE REVIEW

File: _attributes/common-attributes.adoc [hunk 1]
```diff
@@ -101,13 +101,13 @@ endif::[]
 //distributed tracing
 :DTProductName: Red Hat OpenShift distributed tracing
 :DTShortName: distributed tracing
-:DTProductVersion: 2.3
+:DTProductVersion: 2.4
 :JaegerName: Red Hat OpenShift distributed tracing platform
 :JaegerShortName: distributed tracing platform
-:JaegerVersion: 1.30.1
+:JaegerVersion: 1.33
```
Comment (frzifus) [inline _attributes/common-attributes.adoc:107]:
```suggestion
:JaegerVersion: 1.34.1
```

**Review by pavolloffay — APPROVED** [review]
LGTM, 

but the @frzifus comments need to be fixed.

**Review by frzifus — APPROVED** [review]

**Review by iblancasa — APPROVED** [review]

**Review by jc-berger — COMMENTED** [review]
I consulted the openshfit checklist when peer reviewing this PR: https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.mgz8v4w1kg86

File: modules/distr-tracing-rn-new-features.adoc [hunk 2]
```diff
@@ -13,17 +13,48 @@ Result – If changed, describe the current user experience.
 
 This release adds improvements related to the following components and concepts.
 
+== New features and enhancements {DTProductName} 2.4
+
+This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+This release also adds support for auto-provisioning certificates using the Red Hat Elasticsearch Operator.
+
+* Self-provisioning, that is using the {JaegerName} Operator to call the Red Hat Elasticsearch Operator during installation, is fully supported with this release.
```
[the remark this answers is not in the document]
Comment (jc-berger) [reply modules/distr-tracing-rn-new-features.adoc:22]: this is a bit wordier but maybe clearer?

`Self-provisioning means using the  {JaegerName} Operator to call the Red Hat Elasticsearch Operator during installation. Self provisioning is fully supported with this release.`

**Review by jc-berger — APPROVED** [review]
/lgtm

File: modules/distr-tracing-rn-new-features.adoc [hunk 3]
```diff
@@ -13,17 +13,48 @@ Result – If changed, describe the current user experience.
...
+This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+This release also adds support for auto-provisioning certificates using the Red Hat Elasticsearch Operator.
+
+* Self-provisioning, which means using the {JaegerName} Operator to call the Red Hat Elasticsearch Operator during installation. Self provisioning is fully supported with this release.
+* Creating the Elasticsearch instance and certificates first and then configuring the {JaegerShortName} to use the certificate is a Technology Preview for this release.
+
+[NOTE]
+====
+When upgrading to Elasticsearch 2.4, the Operator recreates the Elasticsearch instance, which might take five to ten minutes.
```
Comment (jkandasa) [inline modules/distr-tracing-rn-new-features.adoc:27]: @JStickler 
`When upgrading to Elasticsearch 2.4` should be as `When upgrading to distributed tracing 2.4`.
Also needs to specify that the recreation period is a complete down time.

**Pull request merged**